PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,15 @@
+A brief description of the changes in this PR:
```

Comment:
Could be a simple Fixes and Changes to keep this clean, such as

# Fixes:

# Changes:

And see how it looks like here: https://github.com/0chain/0chain/pull/1023

Reply:
I've updated the format to use headings and section separator.